packages/lesswrong/components/tagging/TagPageButtonRow.tsx

```diff
@@ -97,32 +98,52 @@ const TagPageButtonRow = ({tag, editing, setEditing, className, classes}: {
       e.preventDefault();
     }
   }
-  
+
+  const showEdit = !editing
+  const noEditKarmaTooLow = currentUser && currentUser.karma < tagMinimumKarmaPermissions.edit
+  const noEditNotAuthor = currentUser && !currentUser.isAdmin && tag.authorOnly && tag.userId !== currentUser._id
+  const canEdit = showEdit && !noEditKarmaTooLow && !noEditNotAuthor
+
   const editTooltip = <>
+    {noEditNotAuthor && <>
+      <div>
+      This article can only be edited by the author or an admin, please comment in the discussion to suggest changes
+    </div>
+    <br />
+    </>}
+    {noEditKarmaTooLow && <>
+      <div>
+      You must have at least {tagMinimumKarmaPermissions.edit} karma to edit this topic
```

Comment:
unrelated but think this will be helpful as I couldn't work out why the edit button wasn't appearing to begin with